extension/docs/HowToRun.md

```diff
@@ -18,21 +18,61 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Environment variables
+## Environment variable
+
+Extension module requires 1 environment variable to start. This environment variable name is `ADYEN_INTEGRATION_CONFIG` and it must contain settings as attributes in a JSON structure.
+
+```json
+{
+  "commercetools": {
+    "commercetoolsProjectKey1": {
+      "clientId": "xxx1",
```

Comment:
better to remove numeric on credentials part. It is obvious adyen and commercetools credentials are different

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/542/commits/d42f7d4081ca1c82ece8877303c4badc98ac1cb8